controllers/concerts.controller.js

```diff
@@ -70,4 +70,59 @@ exports.deleteItem = async (req, res) => {
     } catch (err) {
         res.status(500).json({ message: err });
     }
+};
+
+exports.getPerformer = async (req, res) => {
+    try {
+        const performer = req.params.performer;
+        const getPerformer = await Concert.find({ performer: performer });
```

Comment:
Nazwa `getPerformer` sugeruje, że jest to nazwa akcji, a nie jej rezulat. Zmień stałą perfomer wyżej na np. `performerId`, a `getPerfomer` na `performer`, aby nazwy zmiennych odzwierciedlały co ten kod robi. Jest to zasada samodokumentującego się kodu.